network.go

```diff
@@ -27,8 +29,8 @@ type Network struct {
 	instanceID uint64
 	logger     *log.Entry
 
-	globalCloser chan interface{}
-	fatalError   chan error
+	stopper        chan interface{}
+	startStopMutex sync.Mutex
```

Comment:
not using that?

Reply:
I was playing around with the ability to stop and then restart the network but abandoned it for now since it would dramatically increase the complexity with little to no gain compared to just creating a new network.